server/resources/CampaignResources.py

```diff
@@ -0,0 +1,62 @@
+from flask_restful import Resource, reqparse
+from models.CampaignModel import CampaignModel
+from models.ProspectModel import ProspectModel
+from flask_jwt_extended import jwt_required, get_jwt_identity
+from flask import json
+
+parser = reqparse.RequestParser()
+parser.add_argument('name')
+parser.add_argument('prospect_ids', action='append')
+parser.add_argument('campaign_id')
+
+class NewCampaign(Resource):
+    @jwt_required
+    def post(self):
+        data = parser.parse_args()
+
+        new_campaign = CampaignModel(
+            name = data['name'],
```

Comment:
I usually make a decorator in python to throw 400 errors if parameters are not given - it'd be nice to handle if name is not given in the request